manifests/profile/hathitrust/ingest_jobs.pp

```diff
@@ -0,0 +1,178 @@
+    'mail rights load summary':
+      command => "/bin/mail -s \"Rights load summary\" ${recipient} < ${rights_log};\
+ mv ${rights_log} ${feed_log}/populate_rights_`date +\"\\%Y\\%m\\%d\"`.log",
+      hour    => 23,
+      minute  => 59;
+
+    # summary of previous day's ingest activity
+    'ingest summary':
+      command => "/bin/bash ${feed_home}/bin/feed.daily/ingest_summary.sh",
+      hour    => 0,
+      minute  => 5;
+
+    # run daily tasks: update grin, get bibrecords, queue new material, deposit rights
+    'daily tasks':
+      command => "for script in ${feed_home}/bin/feed.daily/enabled/*pl; do perl -I${feed_home}/lib \$script; done",
+      hour    => 3,
+      minute  => 0;
+
+    # copy crms renewals from latte-1
+    'crms renewals':
+      command => "/usr/bin/rsync ${crms_renewals_source} ${crms_renewals_dest}",
+      hour    => 4,
+      minute  => 5;
+
+    'copy google rejects list':
+      command     => "${feed_home}/bin/feed.daily/copy_rejects.sh",
+      environment => "MAILTO=${joined_dcu_recipients}",
+      hour        => 3,
+      minute      => 5;
+
+    ## WEEKLY JOBS
+
+    'get brittle books data':
+      command => "${feed_perl}/feed.weekly/get_brittle_books_data.pl 2>&1 > /dev/null",
+      weekday => 1,
+      hour    => 8,
+      minute  => 0;
+
+    'generate ingest logs':
+      command =>  "${feed_perl}/feed.weekly/generate_logs.pl",
+      weekday => 1,
+      hour    => 1,
+      minute  => 0;
+
+    'generate ingest reports':
+      command => "${feed_perl}/feed.weekly/ingest_reporting.pl",
+      weekday => 1,
+      hour    => 2,
+      minute  => 0;
+
+    'repository statistics':
+      command => "/bin/bash ${feed_home}/bin/feed.weekly/repostat.sh",
+      weekday => 1,
+      hour    => 0,
+      minute  => 0;
+
+    'audit statistics':
+      command => "/bin/bash ${feed_home}/bin/feed.weekly/auditstat.sh",
+      weekday => 1,
+      hour    => 0,
+      minute  => 5;
+
+    ## MONTHLY JOBS
+    ########## MONTHLY ############
+
+    # TODO: upload grin_gfv script
+    # Set volumes that are VIEW_FULL on GRIN but ic or und in the rights DB to
+    # pdus/gfv, and reset volumes that are pdus/gfv but not VIEW_FULL on GRIN to
+    # their bib-determined rights
+
+    # not sure if this is still needed: "disposition 0 volumes"
+    'grin reports':
+      command  => "${feed_perl}/feed.monthly/grin_reports.pl",
+      monthday => 8,
+      hour     => 1,
+      minute   => 1;
+
+    'grin gfv':
+      command     => "${feed_perl}/feed.monthly/grin_gfv.pl",
+      environment => $base_env + [ "HTFEED_CONFIG=${rights_db_config}" ],
+      monthday    => 29,
+      hour        => 1,
+      minute      => 1;
+
+    'all sources':
+      command  => "/bin/bash ${feed_home}/bin/feed.monthly/all_sources.sh",
+      monthday => 1,
+      hour     => 2,
+      minute   => 40;
+
+    'generate dpla log':
+      command  => '/bin/bash /htapps/babel/stats/bin/generate_dpla_log.sh',
```

Comment:
Unlikely to move, but all of the other paths in this section are relative to some variable.